core/db/curibio/alembic/versions/5d220341bf80_pulse3d_0_27_1.py

```diff
@@ -0,0 +1,24 @@
+"""pulse3D 0.27.1
+
+Revision ID: 5d220341bf80
+Revises: 705b8a7903c8
+Create Date: 2022-10-20 13:51:12.789508
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "5d220341bf80"
+down_revision = "705b8a7903c8"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("INSERT INTO pulse3d_versions (version) VALUES ('0.27.1')")
```

Comment:
will you add "INSERT INTO pulse3d_versions (version, state) VALUES ('0.27.1', 'external')"

Reply:
I will add that, what does the external specify?

Reply:
Did you mean replace the query ?

Reply:
We've changed it so that only versions marked 'external' will be shown to customers in production so that they don't get cluttered with all the versions we release during a sprint cycle.. this is what allows us to show which mark versions as 'testing' in the test cluster and allow us to use all the versions before getting released

Reply:
can we also delete version 0.27.0, since it has a bug and people shouldn't use it